src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -42,6 +42,9 @@ public class RestRequest {
   // min backoff in milli before we retry due to transient issues
   private static final long minBackoffInMilli = 1000;
 
+  // min backoff in milli for login/auth requests before we retry
+  private static final long minLoginBackoffInMilli = 4000;
```

Comment:
Why 4? As I see in the doc and in python implementation, first sleep time is 1s?

Reply:
When we were implementing the new retry strategy for all the drivers, some questions came up around this and whether we could reach a default timeout of 5 minutes with the default 7 retries if we start at 1. You can see the discussion here: https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/575
So this is inline with the implementation for all the drivers.